### PR TITLE
Adding properties to channel resource in v1.0

### PR DESCRIPTION
**Pull request by AkJo**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f7c8022](https://github.com/AkJo/microsoft-graph-docs/pull/4236/commits/f7c80223d7a2f61b8dd9d78ea6ec1b127b23f18d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/channel.md](https://github.com/AkJo/microsoft-graph-docs/blob/master/api-reference/v1.0/resources/channel.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/AkJo/microsoft-graph-docs/blob/master/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903250512357421-4236%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GAt0BkR7YSqESOD3b36EcelXNay2c%2BNpXryXT6RnWHU%3D&st=2019-03-25T05%3A11%3A24Z&se=2019-04-25T05%3A16%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by nkramer** [comment]
dup of https://github.com/microsoftgraph/microsoft-graph-docs/pull/4323

File: api-reference/v1.0/resources/channel.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ Examples can be "Friday Team Lunch" channel, and "Architecture Discussion" chann
 |description|String|Optional textual description for the channel.|
 |displayName|String|Channel name as it will appear to the user in Microsoft Teams.|
 |id|String|The channels's unique identifier. Read-only.|
+|email|string| The email address for sending messages to the channel. Read-only.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/channel.md]: Please also update the JSON representation to reflect the new properties, and any examples if applicable.

File: concepts/changelog.md [hunk 2]
```diff
@@ -61,6 +61,12 @@ Debut of places API to provide rich details for locations in apps.
 |:---|:---|:---|
 |Addition|beta|Added the **expirationDatetime** and **password** properties to the [createLink](/graph/api/driveitem-createlink?view=graph-rest-beta) action. |
 
+### Microsoft Teams APIs
```
Comment (Lauragra) [inline concepts/changelog.md]: An entry for this change was added to the changelog under April, so we should remove this or the April one, depending on when this change was introduced.

**Comment by clearab** [comment]
This change has been incorporated previously.

**Pull request closed without merging**